lib/requests/diaryFoods.js

```diff
@@ -115,6 +117,37 @@ const styleRemainingCalorieCount = (meal, remainingCalories) => {
   }
 }
 
+const calculateDiaryTotalCaloriesConsumed = () => {
```

Comment:
I was able to resolve the refactoring error from today's lesson and we could probably refactor this with one of the approaches we made to clean up this code.